### PR TITLE
Added third paragraph

### PR DESCRIPTION
**Pull request by purna441**
This is third paragraph

### CODE REVIEW

File: demo.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 Hello Simplelearn
 This is second paragraph
+This is third paragraph
```
Comment (purna441) [inline demo.txt]: what is purpose of this?

Comment (purna441) [reply demo.txt]: this is demo purpose

Comment (purna441) [reply demo.txt]: ok

**Review by purnapinninti — APPROVED** [review]
approved

**Pull request merged**